src/jit/lsra.cpp

```diff
@@ -2958,12 +3274,23 @@ LinearScan::buildRefPositionsForNode(GenTree *tree,
 #ifdef DEBUG
     if (VERBOSE)
     {
-        JITDUMP("at start of tree, stack is : [ ");
-        for (int i=0; i<stack->Height(); i++)
+        JITDUMP("at start of tree, map contains: { ");
+        bool first = true;
+        for (auto kvp : operandToLocationInfoMap)
```

Comment:
I'm not sure that the reference form would compile: the relevant iterators only produce values.